seed/seed16.go

```diff
@@ -465,3 +465,7 @@ func (s *seed16) Iter(f func(sn *Snap) error) error {
 	}
 	return nil
 }
+
+func (s *seed16) LoadAutoImportAssertion(commitTo func(*asserts.Batch) error) error {
+	return fmt.Errorf("Auto-import from seed is not supported")
```

Comment:
nitpick: our error messages always start with a small caps letter.

Reply:
yes, error strings should start with lower case and also be formulate as "cannot..." whenever possible. This is not a case where the latter applies, anyway per my other comment not sure we even need to have this method here.

Reply:
This is now removed completely as I moved `LoadAutoImportAssertions` into own interface.